cypress.config.ts

```diff
@@ -22,6 +22,7 @@ export default defineConfig({
       APP_AUTH0_CLIENT_ID: getEnv('APP_AUTH0_CLIENT_ID'),
       APP_API_ENDPOINT: getEnv('APP_API_ENDPOINT'),
       APP_HASURA_ADMIN_SECRET: getEnv('APP_HASURA_ADMIN_SECRET'),
+      http_proxy: getEnv('HTTP_PROXY'),
```

Comment:
Is it used?

Reply:
Yeah, I had an issue with that and I needed to add it for e2e to work for me.

Reply:
But I didn't find it in the code